.github/workflows/docker_img.yaml

```diff
@@ -43,6 +43,6 @@ jobs:
             - name: Scan for vulnerabilities
               uses: crazy-max/docker-scan-action@master
               with:
-                  image: connectedhomeip/chip-build${{ matrix.img }}:0.5.28
+                  image: connectedhomeip/chip-build${{ matrix.img }}:latest
```

Comment:
please do not use ':latest'. 

Reasoning for this is that we are  creating branches that stop changing and we want to keep referencing things that 'were working on branches' rather than a moving target.